receiver/azureeventhubreceiver/config.go

```diff
@@ -25,12 +25,13 @@ var (
 )
 
 type Config struct {
-	Connection    string        `mapstructure:"connection"`
-	Partition     string        `mapstructure:"partition"`
-	Offset        string        `mapstructure:"offset"`
-	StorageID     *component.ID `mapstructure:"storage"`
-	Format        string        `mapstructure:"format"`
-	ConsumerGroup string        `mapstructure:"group"`
+	Connection               string        `mapstructure:"connection"`
+	Partition                string        `mapstructure:"partition"`
+	Offset                   string        `mapstructure:"offset"`
+	StorageID                *component.ID `mapstructure:"storage"`
+	Format                   string        `mapstructure:"format"`
+	ConsumerGroup            string        `mapstructure:"group"`
+	ApplySemanticConventions bool          `mapstructure:"apply_semantic_conventions"`
```

Comment:
Can you add a new section in the readme describing this new setting.

Reply:
Added in [7110692](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/35357/commits/71106927d656b95e51f2c4c4c53d1c6df90b37b1)